traffic_ops/app/lib/API/Profile.pm

```diff
@@ -208,6 +217,7 @@ sub create {
 			description => $description,
 			cdn         => $cdn,
 			type        => $type,
+			ccr_routing_disabled => $ccr_routing_disabled,
```

Comment:
you can just set this to 

ccr_routing_disabled => defined($params->{routingDisabled}) ? $params->{routingDisabled} : 0;